support-lib/objc/DJIError.h

```diff
@@ -30,6 +30,6 @@ namespace djinni {
     ::djinni::throwUnimplemented(__PRETTY_FUNCTION__, msg);
 
 #define DJINNI_TRANSLATE_EXCEPTIONS() \
-    catch (const std::exception & e) { \
+    catch (__unused const std::exception & e) { \
```

Comment:
Wouldn't it be better to simply delete the name?  It is truly unused, and removing the name is a more standard way to do it in C++.